src/WorkOS.net/Services/Passwordless/_interfaces/CreatePasswordlessSessionOptions.cs

```diff
@@ -19,6 +19,12 @@ public class CreatePasswordlessSessionOptions : BaseOptions
         [JsonProperty("type")]
         public PasswordlessSessionType Type { get; set; }
 
+        /// <summary>
+        /// The location that the user will be redirected to after authenticating.
```

Comment:
Nit: can indicate this parameter is optional.

Reply:
Ah, for sure! I'll add that into the comment, thanks!

Reply:
Probably not in this PR, but we should update the SDK with support for [nullable reference types](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references).

Reply:
whoops, making an adjustment because my comment broke something 🤦